Automattic/wp-calypso
# Growth: Minimal stepper onboarding flow

Closes Automattic/growth-foundations#70

## Proposed Changes

Adds a minimal `/setup/onboarding-pm-dev` flow that should be identical to `/start` -> with "log in", "domains", "plans", "checkout", steps.

### Not covered under this PR

- Update the term toggle to 1y/2y
-  Processing step: the progress bar is not filled-up properly
-  Processing step: the progress bar should attach to the top as https://github.com/Automattic/wp-calypso/pull/77801#discussion_r1226681785 said
-  In the main onboarding flow, there is a Business upsell shown after picking the Premium plan. We should replicate that here.
-  Confirm that is_domain_mapping and is_domain_transfer are properly set for calypso_signup_completion event.


## Testing Instructions



* Go to `/setup/onboarding-pm-dev` and confirm the flow end to end. It should behave identically to `/start`

## Pre-merge Checklist
